### ISSUE TITLE
Display appropriate property names

### ISSUE THREAD

**Issue by nikolagsiderov** [nikolagsiderov/APPartment#287]
At the moment, all ViewModel properties' names for all objects display the property name by default. This means that in `UserPostViewModel.cs` the property for `Name` display as 'Name' in the field when you go to the login/register pages.

We want to display appropriate namings here. So 'Name' for UserPostViewModel is not enough... We want to display 'Username' in the field. Check out attached shot in this issue's full details for visualization.

With the ViewModels introduced back in iteration 2 by Koleto, we can now do this. So, let's do it!

**Technical part:**
You will go through the ViewModels and will add an attribute `[Display(Name = "<Display name>")]`. There might be cases where the view model will not explicitly have the property you are searching for... For instance, in `UserPostViewModel` you will have to set a display name for `Name` property, but the `Name` property is not a derived property of `UserPostViewModel`, it's an inherited one from `PostViewModel`. So what you will to is `override` the property and set a `[Display(Name = "<Username")]` attribute.

View model properties to set display names:
1. In `UserPostViewModel` set `Name` with `[Display(Name = "Username")]`
2. In `UserPostViewModel` set `ConfirmPassword` with `[Display(Name = "Confirm password")]`
3. In `HomePostViewModel` set `Name` with `[Display(Name = "Home name")]`
4. In `HomePostViewModel` set `ConfirmPassword` with `[Display(Name = "Confirm password")]`
5. In `HomeSettingPostViewModel` set `RentDueDateDay` with `[Display(Name = "Rent due date day")]`
6. In `HomeSettingPostViewModel` set `HomeName` with `[Display(Name = "Home name")]`
7. In `SurveyPostViewModel` set `IsCompleted` with `[Display(Name = "Completed")]`
8. In `SurveyDisplayViewModel` set `IsCompleted` with `[Display(Name = "Completed")]

Update 21/03 from Koleto here:
Also on list, set following display names:
1. In `InventoryPostViewModel` set `IsSupplied` with `[Display(Name = "Supplied")]`
2. In `IssuePostViewModel` set `IsClosed` with `[Display(Name = "Closed")]`
3. In `ChorePostViewModel` set `IsDone` with `[Display(Name = "Done")]`
4. In `HygienePostViewModel` set `IsDone` with `[Display(Name = "Done")]`

Check out if you modifications are applied.

**Comment by nikolagsiderov** [comment]
Everything is very good here! Nice!

There was an issue in the Chores module, so @fr0wBtW did not add the display attribute for `IsDone` property in Chore view models. The issue is now fixed, so @fr0wBtW can now add it.
Also, in the description I have added in the list of view models to add display attributes in `SurveyDisplayViewModel` and `HomeSettingsPostViewModel` for `HomeName` property, but they are not being properly consumed in the UI atm, so @fr0wBtW will skip these ones here.
